pkg/api/routes.go

```diff
@@ -107,6 +107,9 @@ func (a *NodeApi) routes(opts *RunOptions) (chi.Router, error) {
 			r.Get("/height/{id}", wrapper(a.BlockHeightByID))
 			r.Get("/at/{height}", wrapper(a.BlockAt))
 			r.Get("/{id}", wrapper(a.BlockIDAt))
+			// Finalization.
+			r.Get("/height/finalized", wrapper(a.FinalizedHeight))
+			r.Get("/headers/finalized", wrapper(a.FinalizedHeader))
```

Comment:
Below you have `/headers` subroute. Suggest move this endpoint to the corresponding subroute.